internal/service/cloudbackupsnapshotrestorejob/resource_cloud_backup_snapshot_restore_job_test.go

```diff
@@ -18,11 +18,13 @@ const (
 	dataSourceName = "data.mongodbatlas_cloud_backup_snapshot_restore_job.test"
 )
 
-var clusterRequest = acc.ClusterRequest{
-	CloudBackup: true,
-	ReplicationSpecs: []acc.ReplicationSpecRequest{
-		{Region: "US_WEST_2"},
-	},
+func clusterRequest() *acc.ClusterRequest {
```

Comment:
what was the problem to have to use this? was the var being mutated?

Reply:
oh, the random name for cluster being reused?

Reply:
Yes. The same "instance" was being re-used in the different tests.
Therefore, the cluster_name was the same for all tests and we got the DUPLICATE_CLUSTER_NAME error